tests/conftest.py

```diff
@@ -59,7 +59,10 @@ def getoption_side_effect(name, default=None):
     mocked_config.rootdir = py.path.local('/path/to')
     mocked_config.trace = TagTracer().get('root')
     mocked_config.pluginmanager = mock.Mock()
-    mocked_config.option = mock.Mock()
+    mocked_config.option = mock.create_autospec(Config)
```

Comment:
This line reproduces the issue by making `pconfig.option` as restricted as it would be in normal operation. This is why the required fields must now be set explicitly below.